test/ip-addresses.ts

```diff
@@ -13,138 +13,7 @@
 // limitations under the License.
 
 import t from 'tap';
-import {
```

Comment:
The result of inlining the parse IP address logic is that you can't easily communicate the expected behavior through unit tests like this.